.gitignore

```diff
@@ -1,3 +1,6 @@
 /target
 /Cargo.lock
 .vscode
+.env
+__pycache__
+*.so
```

Comment:
Entirely cosmetic but we typically add one empty line to end of file.

Reply:
cosmetic haha

Reply:
Actually, in some edge cases it can lead to unnecessary git conflicts when several people changes the last lines of the file. So we have this rule of always ending all files with an empty line